src/.gitignore

```diff
@@ -9,6 +9,10 @@ callgrind.out.*
 # Apple debuging symbol files
 *.dSYM
 
+# .clangd, as this is host machine specific. See .clangd-gen.py to genererate a new
+# .clangd configuration file instead
+.clangd
```

Comment:
Use a single, global, .gitignore

Reply:
Fine, I have moved the content of `src/.gitignore` to `.gitignore`